tests/blackbox/stratis_cli_cert.py

```diff
@@ -148,6 +162,42 @@ def test_pool_create(self):
             True,
         )
 
+    def test_pool_create_encrypted(self):
+        """
+        Test creating an encrypted pool.
+        """
+        with KernelKey("test-password") as key_desc:
+            pool_name = p_n()
+            self.unittest_command(
+                [
+                    STRATIS_CLI,
+                    "pool",
+                    "create",
+                    "--key-desc",
+                    key_desc,
+                    pool_name,
+                    StratisCertify.DISKS[0],
+                ],
+                0,
+                True,
+                True,
+            )
+
+    def test_pool_create_encrypted_with_cache(self):
+        """
+        Test creating an encrypted pool with cache.
+
+        This should fail.
+        """
+        with KernelKey("test-password") as key_desc:
+            pool_name = make_test_pool(StratisCertify.DISKS[0:2], key_desc)
+            self.unittest_command(
+                [STRATIS_CLI, "pool", "init-cache", pool_name, StratisCertify.DISKS[2]],
+                1,
+                True,
```

Comment:
For some reason stderr comes before standout in arg order and I think it's stderr that we should expect to be non-empty.

Reply:
Okay I assumed it would be the opposite! I'll fix that.

Reply:
Should be fixed now.